learners/setup.md

```diff
@@ -21,7 +21,7 @@ training_site: https://carpentries.github.io/instructor-training
 3. Please read the following before the workshop begins:
 
 - ["The Science of Learning"](./files/papers/science-of-learning-2015.pdf)
-- ["The Carpentries Annual Report"](https://carpentries.org/files/reports/AnnualReport2022.pdf)
+- ["The Carpentries Annual Report"](https://carpentries.org/reports/annual-reports/2023AnnualReport.pdf)
```

Comment:
```suggestion
- "The Carpentries Annual Report" from [The Carpentries Impact page](https://carpentries.org/about-us/impact/)
```

per #1755 recommendation to link the page instead of having to update annually.


Maybe we should add "most recent" in somewhere to make it more clear?